modules/core/src/scenegraph/nodes/scenegraph-node.js

```diff
@@ -18,7 +18,11 @@ export default class ScenegraphNode {
     this._setScenegraphNodeProps(props);
   }
 
-  delete() {}
+  delete() {
```

Comment:
This should go into `Group.delete()` as that class manages `children`

By convention we also protect against multiple deletes.

```
class Group ... {
  delete() {
    this.children.forEach(child => child.delete());
    this.children = [];
    super.delete();
  }
}
```
The implementation here in the base class should be empty 

```
  delete() {}
```

The `ModelNode` needs to reimplement `delete()` to call `model.delete()`